default.json5

```diff
@@ -133,6 +133,13 @@
       // 0.8.0 needs golang 1.22
       "allowedVersions": "< 0.8.0",
       "enabled": true
+    },
+    {
+      "groupName": "metallb",
+      "matchPackagePatterns": ["^github.com/metallb/frr-k8s"],
+      // 0.0.12 needs golang 1.22
+      "allowedVersions": "< 0.0.12",
+      "enabled": true
```

Comment:
I think we want this in the stable.json as well to limit this bump on the 18.0-frx branches

Reply:
Ahh you are right, we intentionally don't want to bump project external deps on stable, so it is already disabled. Sorry for the mixup.